src/components/PaliaClock.vue

```diff
@@ -2,6 +2,7 @@
 import SvgDonutArc from './SvgDonutArc.vue';
 import { computed, onMounted, onUnmounted, ref } from 'vue';
 
+const PAGE_TITLE = 'Palia Clock';
```

Comment:
This is the same title from `index.html`

https://github.com/SharkyTheWhite/palia-clock/blob/81e1f5bee97246b1da9120341069e0bafb911c4b/index.html#L7

I wonder if there's a nicer way to share the same value.

Reply:
In theory, we could define the base title on project level and inject it into the index.html within the Vite build step. But that is not worth the effort at this moment I think. Especially since it will be overwritten by the very feature we are adding here. Good DRY thinking though - might come up again if we ever go to localize the app or add multiple pages with vue-router.